src/themes/default/mixins.scss

```diff
@@ -7,11 +7,10 @@
 */
 @mixin a() {
   a {
-    color: $link-color;
+    color: $gray-base;
     text-decoration: $link-decoration;
 
     &:hover {
-      color: $link-color-hover;
       text-decoration: $link-decoration-hover;
```

Comment:
can't you just set the variables ?

Reply:
What do you mean exactly?

Reply:
As discussed, I set the variables in examples.scss instead of reoving it here